app/views/hub/users/edit.html.erb

```diff
@@ -36,17 +43,16 @@
 
         <p id="current-role"><%= [user_role(@user), user_group(@user)].compact.join(", ") %></p>
 
-        <%= render('components/molecules/reveal', title: t(".reassign")) do %>
-          <ul class="list--bulleted">
-            <li><%= link_to t("general.admin"), edit_role_hub_user_path({user: {role: AdminRole::TYPE}}) %></li>
-            <li><%= link_to t("general.client_success"), edit_role_hub_user_path({user: {role: ClientSuccessRole::TYPE}}) %></li>
-            <li><%= link_to t("general.greeter"), edit_role_hub_user_path({user: {role: GreeterRole::TYPE}}) %></li>
-            <li><%= link_to t("general.coalition_lead"), edit_role_hub_user_path({user: {role: CoalitionLeadRole::TYPE}}) %></li>
-            <li><%= link_to t("general.organization_lead"), edit_role_hub_user_path({user: {role: OrganizationLeadRole::TYPE}}) %></li>
-            <li><%= link_to t("general.site_coordinator"), edit_role_hub_user_path({user: {role: SiteCoordinatorRole::TYPE}}) %></li>
-            <li><%= link_to t("general.team_member"), edit_role_hub_user_path({user: {role: TeamMemberRole::TYPE}}) %></li>
-          </ul>
-        <% end %>
+        <div style="width: 375px">
+          <%= render('components/molecules/reveal', title: t(".reassign")) do %>
+            <ul class="list--bulleted">
+              <% User.roles.each do |role_type| %>
```

Comment:
This limits the display of the roles you can change to to the ones that the current user is allowed to manage. Before, any user could click to try to change a User they could see to be an admin, even if they did not have permissions to actually do that.